src/webgl/interaction.js

```diff
@@ -175,43 +181,84 @@ p5.prototype.orbitControl = function(
     // if mouseLeftButton is down, rotate
     // if mouseRightButton is down, move
     if (this._mouseWheelDeltaY !== 0) {
-      executeZoomProcess = true;
+      accelerateZoomVelocity = true;
       // zoom according to direction of mouseWheelDeltaY rather than value.
-      const mouseWheelSign = (this._mouseWheelDeltaY > 0 ? 1 : -1);
-      deltaRadius = mouseWheelSign * sensitivityZ * mouseZoomScaleFactor;
+      deltaRadius = this._mouseWheelDeltaY * sensitivityZ;
+      deltaRadius *= mouseZoomScaleFactor;
       this._mouseWheelDeltaY = 0;
     }
     if (this.mouseIsPressed) {
       if (this.mouseButton === this.LEFT) {
-        executeRotateProcess = true;
+        accelerateRotateVelocity = true;
         deltaTheta = -sensitivityX * (this.mouseX - this.pmouseX) / scaleFactor;
         deltaPhi = sensitivityY * (this.mouseY - this.pmouseY) / scaleFactor;
       } else if (this.mouseButton === this.RIGHT) {
-        executeMoveProcess = true;
+        accelerateMoveVelocity = true;
         moveDeltaX = this.mouseX - this.pmouseX;
         moveDeltaY = this.mouseY - this.pmouseY;
       }
     }
   }
 
   // interactions
-  // zoom
-  if (executeZoomProcess) {
-    this._renderer._curCamera._orbit(0, 0, deltaRadius);
+
+  // zoom process
+  if (accelerateZoomVelocity) {
```

Comment:
If we initialize `deltaRadius` to 0, maybe we could simplify this section by removing the need for an `accelerateZoomVelocity` flag? And maybe similarly for the rotation/move flags

Reply:
If initialize it to 0 and only process it if it's not 0 then probably don't need the flag... ok, I'll think about it!